### ISSUE TITLE
Error: 400 Bad Request

### ISSUE THREAD

**Issue by dremelts** [Kozea/Radicale#1344]
I am trying to import a large .ics file (about 1300 events exported from NextCloud) into Radicale and I get Error: 400 Bad Request. 

The import process aborts as soon as it hits an error and there is no progress indicator to tell me how many events it processed before encountering the error. 

I'm running Radicale in Docker and the Docker log file only tells me
`[2023-12-13 15:59:58 +0000] [1/Thread-5934 (process_request_thread)] [WARNING] Bad PUT request on '/admin/edf4d0f3-cd85-23d8-eeb5-cd3b73bb8dfa/': Overwritten recurrence with RRULESET`

I have validated the .ics file using [this](https://icalendar.org/validator.html)

How can I determine which event in the .ics file it is getting hung up on?

**Comment by pbiering** [comment]
Radicale is using "vobjects" to validate ICS, can you lookup whether you find in the ICS file the mentioned UUID?

Also I don't know whether radicale supports such bulk import at all, can you split the large file into dedicated event pieces and upload them one-by-one?

**Comment by dremelts** [comment]
Well, I found a very convoluted way to get my ics file into Radicale. I imported it into my mailbox.org calendar and out of about 1300 appointments it said 5 were ignored. (Didn't say which ones though)  Unfortunately, mailbox.org will not connect with Radicale but it _does_ offer its own CalDav server so I was able to connect to the mailbox.org calendar from Evolution on my Linux laptop.  Evolution will also connect to Radicale and so I just copied one calendar to the other. Now Evolution is syncing with Radicale and I can use DAVx5 to sync to my phone calendar (Etar).
